docs/pages/pmd/userdocs/cli_reference.md

```diff
@@ -159,7 +159,8 @@ The tool comes with a rather extensive help text, simply running with `--help`!
                description="Path to a file to which report output is written. The file is created if it does not exist. If this option is not specified, the report is rendered to standard output."
     %}
     {% include custom/cli_option_row.html options="--show-suppressed"
-               description="Causes the suppressed rule violations to be added to the report."
+               description="Causes the suppressed rule violations to be added to the report (xml and html formats only)."
+               languages="html, xml"
```

Comment:
```suggestion
               description="Causes the suppressed rule violations to be added to the report if supported by the report format. See [PMD Report formats](pmd_userdocs_report_formats.html) for details."
```

If we change this here, we should also change it in the code, so that this information is also shown at the command line (`--help`):
https://github.com/pmd/pmd/blob/1acaedc158e733cdd10ca771be97417c55991e37/pmd-cli/src/main/java/net/sourceforge/pmd/cli/commands/internal/PmdCommand.java#L141

```diff
-@Option(names = "--show-suppressed", description = "Report should show suppressed rule violations.")
+@Option(names = "--show-suppressed", description = "Report should show suppressed rule violations if supported by the report format.")
```